{ 
  "text" : "An issue was discovered in Libav 12.3. A read access violation in the in_table_init16 function in libavcodec/aacsbr.c allows remote attackers to cause a denial of service (application crash), as demonstrated by avconv.\n" ,
  "entities" : [ { 
    "eid" : "E1" ,
    "keyid" : "ThreatActor#d0f531219210b35f7261f3d6049ee44c" ,
    "type" : "ThreatActor" ,
    "subType" : "ThreatActor" ,
    "value" : "remote attacker" ,
    "begin" : 125 ,
    "end" : 141 ,
    "canonicalForm" : "remote attacker" ,
    "score" : "0.0"
  }
  , { 
    "eid" : "E2" ,
    "keyid" : "TTP#9c98ef6a7451d6f82ca2194ee23d1acb" ,
    "type" : "TTP" ,
    "subType" : "TTP" ,
    "value" : "denial of service" ,
    "begin" : 153 ,
    "end" : 170 ,
    "canonicalForm" : "denial of service" ,
    "score" : "0.0"
  }
  ,  ] ,
  "relations" : [ { 
    "rid" : "R1" ,
    "type" : "cause" ,
    "predicate" : { 
      "begin" : 142 ,
      "end" : 150 ,
      "surfaceForm" : "to cause"
    }
     ,
    "arguments" : [ { 
      "argnum" : 1 ,
      "begin" : 125 ,
      "end" : 141 ,
      "eid" : "E1" ,
      "canonicalForm" : "remote attacker"
    }
    , { 
      "argnum" : 2 ,
      "begin" : 153 ,
      "end" : 170 ,
      "eid" : "E2" ,
      "canonicalForm" : "denial of service"
    }
    ,  ]
  }
  ,  ]
}
